drivers/ethernet/phy/phy_microchip_ksz8081.c

```diff
@@ -289,20 +293,27 @@ static int phy_mc_ksz8081_reset(const struct device *dev)
 		goto done;
 	}
 
-	/* Wait for 500 ms as specified by datasheet */
-	k_busy_wait(USEC_PER_MSEC * 500);
+	/* Wait for at least 500 us as specified by datasheet */
+	k_busy_wait(1000);
```

Comment:
wrong comment or wrong delay in code

Reply:
I'm not clear how these are wrong- are you saying we should update the comment to something like so:

`/* Datasheet specifies a minimum delay of 500us. Wait 1000us to be safe */`

Reply:
ah -- yeah. Sorry. Both comments confused me for the reason you just described. I think it would be worth clarifying to indeed explain that "just" going with the datasheet recommendation isn't enough, otherwise I feel it can read as a typo. Will remove my block though -- sorry again for the noise!

Reply:
> ah -- yeah. Sorry. Both comments confused me for the reason you just described. I think it would be worth clarifying to indeed explain that "just" going with the datasheet recommendation isn't enough, otherwise I feel it can read as a typo. Will remove my block though -- sorry again for the noise!

No worries, if it doesn't make sense to someone seeing it for the first time then it should probably be changed :)